containerized_ml_models/model_api_7/app.py

```diff
@@ -0,0 +1,24 @@
+# !/usr/bin/env python
+# encoding: utf-8
+import json
+from flask import Flask, request, jsonify
+import pickle
+
+app = Flask(__name__)
+
+
+@app.route('/ping', methods=['GET'])
+def ping():
+    return jsonify({"ping": "success"})
+
+
+@app.route('/predict', methods=['POST'])
+def predict():
+    data = json.loads(request.get_data())
+    model = pickle.load(open('model.pkl', 'rb'))
```

Comment:
In line 18, I am afraid you do not close the file, it can cause some issues. Wouldn't it be better to close it before return?